### PR TITLE
MOOSE Docs Slides

### PR DESCRIPTION
**Pull request by gsgall**
This PR represents the initial version of the MOOSE Docs tutorial slides for Zapdos and is intended to make review and collaboration easier.

### CODE REVIEW

**Comment by csdechant** [comment]
This looks like a good start. One of my main question is can GIF or videos be added to the slides? I know the website can have videos, similar to the [MOOSE Gallery](https://mooseframework.inl.gov/gallery.html) section, but I am unsure if this feature is available for the slide deck.

**Comment by gsgall** [comment]
> This looks like a good start. One of my main question is can GIF or videos be added to the slides? I know the website can have videos, similar to the [MOOSE Gallery](https://mooseframework.inl.gov/gallery.html) section, but I am unsure if this feature is available for the slide deck.

Absolutely we can use mp4 files and/or gifs in the slide deck. I have added an example of both of these in [https://github.com/gsgall/zapdos/blob/devel/doc/workshops/tutorial/content/intro/capability_demo.md](https://github.com/gsgall/zapdos/blob/devel/doc/workshops/tutorial/content/intro/capability_demo.md)

**Comment by csdechant** [comment]
I looked at the Precheck fail and the error was "RPC failed; curl 18 transfer closed with outstanding read data remaining". I am unfamiliar with this type of fail regarding CIVET. Personal I had this error rbefore, but it was when I was doing a git pull with poor internet connect. @gsgall @cticenhour Have you seen "RPC failed" during a MOOSE test before?

**Comment by gsgall** [comment]
@csdechant I just pushed some more recent changes to the slides and it looks like things are building fine so far. Maybe we just got unlucky with that Civet build. 

Also I did update the moose submodule locally, I'll go back and fix this sometime soon.

**Comment by gsgall** [comment]
@csdechant @cticenhour I think the slides are in a solid enough state for the tutorial if we want to use them for that. Please give them a look when you get the chance. I would still like to add more detail in the boundary condition section or at least link to the relevant doc pages but that wouldn't really be useful until #250 is merged and this is something that can be done after the tutorial. 

For the warnings about the size increase on the repo potentially it could be useful to think about a zapdos_large_media submodule if we want to add videos or something like a gallery page in the future.

**Comment by cticenhour** [comment]
Thanks @gsgall. Do you have enough time to address a review if I take a look at this first thing in the morning? 

Re: a large media submodule, we should definitely consider it as well as consider generating any media we can on the fly via scripts (which we do in TMAP8).

**Comment by gsgall** [comment]
> Thanks @gsgall. Do you have enough time to address a review if I take a look at this first thing in the morning?
> 
> Re: a large media submodule, we should definitely consider it as well as consider generating any media we can on the fly via scripts (which we do in TMAP8).

Yeah absolutely, I can do that! 

I also just saw PCs message in the slack so I will update the FENIX section to have a link to our website.

**Comment by gsgall** [comment]
> > Thanks @gsgall. Do you have enough time to address a review if I take a look at this first thing in the morning?
> > Re: a large media submodule, we should definitely consider it as well as consider generating any media we can on the fly via scripts (which we do in TMAP8).
> 
> Yeah absolutely, I can do that!
> 
> I also just saw PCs message in the slack so I will update the FENIX section to have a link to our website.

I think a review from you would be really useful! 

There are probably better ways to achieve the same formatting I have. I suspect my MooseDocs code is a bit hacky

**Comment by csdechant** [comment]
@gsgall The slide doc seems good. My original comment I had during the meeting last Wednesday still holds (about condensing some slides into just one slide), but I think that is just a difference in formatting styles.

**Comment by gsgall** [comment]
> @gsgall The slide doc seems good. My original comment I had during the meeting last Wednesday still holds (about condensing some slides into just one slide), but I think that is just a difference in formatting styles.

Yeah that makes sense, are there specific slides you would to be condensed. If so, I can give that a go with some of them.

File: doc/workshops/tutorial/content/examples/helpful_resources.md
Comment (cticenhour) [inline doc/workshops/tutorial/content/examples/helpful_resources.md:8]:
```suggestion
If you are struggling with an issue, search to see if someone else has run into the same problem. If they haven't, start a new discussion! You're also welcome to start a discussion about how to contribute any functionality you have developed so others can use it, to share new ideas, or even show off new results! 

Finally, if you encounter a replicable bug in Zapdos code or would like to request a new feature be developed, you can visit our issues page:
```

File: doc/workshops/tutorial/content/examples/helpful_resources.md
Comment (cticenhour) [inline doc/workshops/tutorial/content/examples/helpful_resources.md:12]:
```suggestion
We do not have time to develop all requested features, but we are happy to help others get started in developing and contributing the features they need for their work. Reach out! :)
```

File: doc/workshops/tutorial/content/examples/tutorial3.md
Comment (cticenhour) [inline doc/workshops/tutorial/content/examples/tutorial3.md:62]:
```suggestion
### What would happen to the rate of the potential decay as pressure changes?
```

File: doc/workshops/tutorial/content/examples/tutorial3.md
Comment (cticenhour) [inline doc/workshops/tutorial/content/examples/tutorial3.md:66]:
```suggestion
!style halign=left
!alert note
```

I would try using this trick to add a little bit of space between the listing and the alert.

File: doc/workshops/tutorial/content/examples/tutorial3-fenix.md
Comment (cticenhour) [inline doc/workshops/tutorial/content/examples/tutorial3-fenix.md:1]:
```suggestion
# Tutorial 3 - Kinetic Implementation using FENIX
```

File: doc/workshops/tutorial/content/examples/tutorial3-fenix.md
Comment (cticenhour) [inline doc/workshops/tutorial/content/examples/tutorial3-fenix.md:4]: This is so big that on my ultrawide display, the link doesn't even show up. I would drop the size of this.

File: doc/workshops/tutorial/content/examples/tutorial3-fenix.md
Comment (cticenhour) [inline doc/workshops/tutorial/content/examples/tutorial3-fenix.md:57]:
```suggestion
All figures show FENIX results superimposed on figures 2.2.a, 2.2.b, and 2.2.c from [!cite](lieberman1994principles). Results from [!cite](lieberman1994principles) are in black and grey while FENIX results are in color.
```

File: doc/workshops/tutorial/content/examples/tutorial3-fenix.md
Comment (cticenhour) [inline doc/workshops/tutorial/content/examples/tutorial3-fenix.md:84]: I would  re-organize a bit around "verified" and "in-development" activities since there is a lot of word repetition here ("has been verified").

File: doc/workshops/tutorial/content/examples/tutorial3-fenix.md
Comment (cticenhour) [inline doc/workshops/tutorial/content/examples/tutorial3-fenix.md:99]:
```suggestion
- Validation by replicating experimental conditions and data
```

File: doc/workshops/tutorial/content/examples/tutorial4.md
Comment (cticenhour) [inline doc/workshops/tutorial/content/examples/tutorial4.md:7]:
```suggestion
Consider an Argon capacitively coupled plasma (CCP). Utilize the [MOOSE Postprocessor system](syntax/Postprocessors/index.md) to calculate the volume-averaged electron temperature.
```

File: doc/workshops/tutorial/content/examples/tutorial5.md
Comment (cticenhour) [inline doc/workshops/tutorial/content/examples/tutorial5.md:14]:
```suggestion
The current input file assumes that all of the electrons enter the water. What will happen if we change the reflection coefficients at the boundary of the water for the electrons and the electron mean energy?
```

**Review by cticenhour — CHANGES_REQUESTED** [review]

File: doc/workshops/tutorial/content/intro/zapdos.md
Comment (csdechant) [reply doc/workshops/tutorial/content/intro/zapdos.md:31]: Commenting since I didn't mention it on the original tutorial slides. For the plot on that slide, it was microwave interferometry measurements (K. E. Greenberg and G. A. Hebner, J. Appl. Phys. 73 (1993) 8126). For general experimental validation, we should probably reference the [V&V paper](https://www.sciencedirect.com/science/article/abs/pii/S0010465523001820).

File: doc/workshops/tutorial/content/intro/zapdos.md
Comment (csdechant) [reply doc/workshops/tutorial/content/intro/zapdos.md:40]: Those results were for "1D Ion & Electron Electrostatic Fluids While Tracking Electron Mean Energy".

File: doc/workshops/tutorial/content/intro/multi_fluid.md
Comment (csdechant) [reply doc/workshops/tutorial/content/intro/multi_fluid.md:204]: Commenting again because I used this terminology in the original PowePoint Slides. For context, the original slides only briefly show the multi-fluid equation and BC equations as two slide with just equations. The idea being that these students just had an hour and half lecture on the fluid approach to plasma modeling and that this just shows that we currently have 3 sets of difference boundary condition assumptions within Zapdos.

@gsgall While #250 is still under review, the descriptions of these BCs are should be done, so you could 'copy-paste' or summaries those descriptions here.

File: doc/workshops/tutorial/content/examples/tutorial5.md
Comment (csdechant) [reply doc/workshops/tutorial/content/examples/tutorial5.md:7]: @cticenhour correct. The tutorial was design for students to change the reflection coefficient at the boundary of the water (basically, it is Alex's Zapdos paper). The Interface Kernel is discuss here since we walk through the input file with the students and show how two domains could be coupled (e.g. plasma and water).

File: doc/workshops/tutorial/content/bib/workshop.bib
Comment (gsgall) [reply doc/workshops/tutorial/content/bib/workshop.bib:1]: I think that would be fine as well, there wasn't a real reason I put them in their own bib file so I will try and move them. If I run into any issues with this I'll report back.

File: doc/workshops/tutorial/content/intro/multi_fluid.md
Comment (gsgall) [reply doc/workshops/tutorial/content/intro/multi_fluid.md:177]: I typically like the $\partial \Omega$ notation for the boundaries. Does this work for you?

Comment (cticenhour) [reply doc/workshops/tutorial/content/intro/multi_fluid.md:177]: That sounds good - keep it general.

File: doc/workshops/tutorial/content/intro/plasma_models.md
Comment (gsgall) [reply doc/workshops/tutorial/content/intro/plasma_models.md:13]: I went ahead and removed the increased font size on these.

File: doc/workshops/tutorial/content/intro/plasma_models.md
Comment (gsgall) [reply doc/workshops/tutorial/content/intro/plasma_models.md:28]: Increased font size was removed

File: doc/workshops/tutorial/content/media/lineplot_visualization/.DS_Store
Comment (gsgall) [reply doc/workshops/tutorial/content/media/lineplot_visualization/.DS_Store:1]: This is in the .gitignore. I think this was my fault using the `git add -f *` to add the media files

File: doc/workshops/tutorial/content/bib/workshop.bib
Comment (gsgall) [reply doc/workshops/tutorial/content/bib/workshop.bib:1]: Yeah no issues here. I moved the citations.

File: doc/workshops/tutorial/content/examples/tutorial3-fenix.md
Comment (gsgall) [reply doc/workshops/tutorial/content/examples/tutorial3-fenix.md:83]: I separated this into two subsections with lists of what is done and what is under work.

File: doc/workshops/tutorial/content/media/lieberman-particles.png
Comment (gsgall) [reply doc/workshops/tutorial/content/media/lieberman-particles.png:1]: I think this should be doable. I can probably apply a threshold filter to the images to make the blacks as dark as possible.

File: doc/workshops/tutorial/content/intro/multi_fluid.md
Comment (gsgall) [reply doc/workshops/tutorial/content/intro/multi_fluid.md:78]: There is no real reason for this. I just tend to write LaTeX like this. So I can adjust this if needed

File: doc/workshops/tutorial/content/intro/multi_fluid.md
Comment (gsgall) [reply doc/workshops/tutorial/content/intro/multi_fluid.md:27]: I changes this to be `Governing Equations` I think this is a bit more appropriate.

File: doc/workshops/tutorial/content/intro/multi_fluid.md
Comment (gsgall) [reply doc/workshops/tutorial/content/intro/multi_fluid.md:211]: I have added the additional notation to denote that these are applied on the boundaries.

File: doc/workshops/tutorial/content/examples/tutorial3-fenix.md
Comment (gsgall) [reply doc/workshops/tutorial/content/examples/tutorial3-fenix.md:4]: I have adjusted this to 30% let me know if that is a good size.

File: doc/workshops/tutorial/content/examples/tutorial1.md
Comment (gsgall) [reply doc/workshops/tutorial/content/examples/tutorial1.md:47]: I think I need a bit more support on this. Can you show me how this would be best for this slide? @cticenhour

File: doc/workshops/tutorial/content/intro/zapdos.md
Comment (gsgall) [reply doc/workshops/tutorial/content/intro/zapdos.md:38]: I have updated this caption.

File: doc/workshops/tutorial/content/intro/zapdos.md
Comment (gsgall) [reply doc/workshops/tutorial/content/intro/zapdos.md:31]: Added the citation to this page.

File: doc/workshops/tutorial/content/media/lieberman-particles.png
Comment (gsgall) [reply doc/workshops/tutorial/content/media/lieberman-particles.png:1]: Screenshot 2024-06-25 at 11 13 44 PM
Screenshot 2024-06-25 at 11 13 53 PM

This is a quick experiment from this threshold filter. I think some of the labels could be replaced with some sharper ones but the lines look decent. What do you think?

Comment (cticenhour) [reply doc/workshops/tutorial/content/media/lieberman-particles.png:1]: I actually like the previous slightly better, since the FENIX results stood out more. I am OK with scrapping this thought.

File: doc/workshops/tutorial/content/examples/tutorial1.md
Comment (cticenhour) [reply doc/workshops/tutorial/content/examples/tutorial1.md:47]: I do not want to hold this up, and I am too busy writing a proposal tonight, so let's leave it for later edits to this tutorial. This can be resolved.

**Comment by gsgall** [comment]
> > > @gsgall The slide doc seems good. My original comment I had during the meeting last Wednesday still holds (about condensing some slides into just one slide), but I think that is just a difference in formatting styles.
> > 
> > 
> > Yeah that makes sense, are there specific slides you would to be condensed. If so, I can give that a go with some of them.
> 
> In short, any of the slides that was directly adapted from the PowerPoint slides (e.g. the "Computational Plasma Models" and tutorial slides). For the tutorial slides, my preference is to have the main information on the left hand side and the output on the right hand side of the slide. That being said, I am unfamiliar with the MOOSE Doc slide deck, so I am unsure if the slides might look too crowded with that format.

@cticenhour @csdechant 

I think the slides are in a good enough state for the presentation tomorrow. I can work on exploring the formatting preference that Corey has one I have more time to really dig into the [Layout Extension](https://mooseframework.inl.gov/python/MooseDocs/extensions/layout.html) as Casey has suggested. I think this will also be necessary to support the additional context in the boundary condition section since I think some formatting will need to change to accommodate additional text or links to the relevant doc pages once they are up.

**Comment by cticenhour** [comment]
@gsgall I was just typing up a similar message when this appeared 😄 I am happy to resolve and merge so that it can be in with enough time to spare.

**Review by csdechant — APPROVED** [review]

File: doc/workshops/tutorial/content/intro/multi_fluid.md
Comment (cticenhour) [reply doc/workshops/tutorial/content/intro/multi_fluid.md:232]: Let's resolve this for now and add it after #250 is in.

File: doc/workshops/tutorial/content/intro/multi_fluid.md
Comment (cticenhour) [reply doc/workshops/tutorial/content/intro/multi_fluid.md:173]: Resolved for now due to time constraints. Let's re-approach later.

File: doc/workshops/tutorial/content/examples/tutorial2.md
Comment (cticenhour) [reply doc/workshops/tutorial/content/examples/tutorial2.md:57]: Let's resolve for now due to time constraints and re-approach later.

**Review by cticenhour — APPROVED** [review]

**Pull request merged**
